recipe/build.sh

```diff
@@ -1,5 +1,37 @@
 #!/bin/bash
+# Turn the work-folder into GOPATH
+export GOPATH=${SRC_DIR}
+export PATH=${GOPATH}/bin:$PATH
+pushd src/github.com/sylabs/${PKG_NAME}
 
-./configure --prefix="${PREFIX}"
-make
-make install
+# bootstrap go dependencies
+go get -u github.com/golang/dep/cmd/dep
+
+# Git Initialize
+# Apps tend to use git info to create version string
+git init
+git config --local user.email "conda@conda-forge.github.io"
+git config --local user.name "conda-forge"
+
+echo $PKG_VERSION >> .conda_version
+git add .conda_version
+git commit -m "conda build of $PKG_NAME-v$PKG_VERSION"
+git tag v${PKG_VERSION}
```

Comment:
Yes! I have refrained from doing so because the reviews usually ask for stable tar-balls. But I am happy to make the change, and this could in fact simplify *a lot* of the go builds.

Reply:
That best practice was to cutdown on resource usage (bandwidth, repo size, CI time, etc.). I think if we are building our own git repo to make sure the build works correctly, then that best practice is causing more harm than good. Would suggest sending a PR to the docs to clarify the best practice in the context of go packages with some information about why git makes sense for go packages.